doc/nrf/releases/release-notes-2.3.0.rst

```diff
@@ -0,0 +1,1034 @@
+.. _ncs_release_notes_230:
+
+|NCS| v2.3.0 Release Notes
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+|NCS| delivers reference software and supporting libraries for developing low-power wireless applications with Nordic Semiconductor products in the nRF52, nRF53, nRF70, and nRF91 Series.
+The SDK includes open source projects (TF-M, MCUboot, OpenThread, Matter, and the Zephyr RTOS), which are continuously integrated and redistributed with the SDK.
+
+Release notes might refer to "experimental" support for features, which indicates that the feature is incomplete in functionality or verification, and can be expected to change in future releases.
+
+To learn more, see :ref:`software_maturity`.
+
+Highlights
+**********
+
+The following list includes the summary of the most relevant changes introduced in this release.
+
+* Added :ref:`support <software_maturity>` for the following features:
+
+  * nRF7002 Wi-Fi 6 companion IC and nRF7002 DK.
+  * Bluetooth® Low Energy: Period Advertising Sync Transfer.
+    This feature allows Periodic Advertising synchronization data to be transferred over an ACL connection, which gives power saving benefits for energy constraint devices.
+    This is a key feature for the `Auracast™`_ Assistant.
+  * nRF Cloud Location Services Wi-Fi® feature support in the :ref:`lib_location` library and demonstrated in :ref:`asset_tracker_v2`, :ref:`modem_shell_application`, and :ref:`lwm2m_client`.
```

Comment:
I don't think this is accurate. Wi-Fi support for Location library and Modem shell sample has been there for quite some time with ESP8266. Even with nRF7002EK, the support was in 2.2.0 already. It's true that nRF7002EK support has been added to ATv2 in 2.3.0. I guess same applies to LwM2M.
I don't know how accurate we want to be but it would be nice to get a comment on this from the one that provided the sentence to the highlights.

Reply:
This has been now changed to:
Location library now supports sending both Wi-Fi and cellular location data to nRF Cloud location services for improved location accuracy. This is demonstrated in Asset Tracker v2, Modem Shell, and LwM2M client.

Reply:
Ok, I guess that's good if the intention is to highlight the feature where we send both Wi-Fi and cellular location data to the cloud at the same time, instead of doing two independent requests.